org.gnome.Hamster.yaml

```diff
@@ -0,0 +1,52 @@
+---
+app-id: org.gnome.Hamster
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: hamster
+finish-args:
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--share=ipc'
+  - '--filesystem=xdg-documents'
+  - '--own-name=org.gnome.Hamster'
```

Comment:
```suggestion
```
You don't need this it's implicit with the app-id

Reply:
removed